src/toast-notification/classes.scss

```diff
@@ -60,13 +55,50 @@
 
     .iui-toast-entering,
     .iui-toast-entered {
-      animation: openAnimation 240ms cubic-bezier(0.175, 0.885, 0.32, 1.175) both;
+      animation: openAnimation 240ms cubic-bezier(0.175, 0.885, 0.32, 1.175)
+        both;
```

Comment:
I would revert this. Looks weird with `both` in new line (even if prettier disagrees).